### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by lusekayz**
This is my first Microverse Project.

I buit it using HTML and CSS

### CODE REVIEW

File: .hintrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "connector": {
+      "name": "local",
+      "options": {
+        "pattern": ["**", "!.git/**", "!node_modules/**"]
+      }
+    },
+    "extends": ["development"],
+    "formatters": ["stylish"],
+    "hints": [
+      "button-type",
```
[the remark this answers is not in the document]
Comment (lusekayz) [reply .hintrc:11]: I use a Token Access Token on Github. Maybe I am missing something. I ran the linters and did not encounter an errors before pushing to my branch to Github

[the remark this answers is not in the document]
Comment (Wes-Isaac) [reply .hintrc:11]: great job @lusekayz

**Comment by lusekayz** [comment]
All linters have passed the test. I added the .github to the commited files.

**Comment by Wes-Isaac** [comment]
hi @lusekayz please open the pull request I see that you've merged it. Kindly don't close pull requests before approval.

**Comment by lusekayz** [comment]
@Wes-Isaac. Where do I open the pull request, I thought all was done.  I am sorry

**Comment by Wes-Isaac** [comment]
It's fine I will aprrove this project but please in the future don't merge a branch before approval

**Comment by lusekayz** [comment]
@Wes-Isaac can you assist with this??

**Review by Wes-Isaac — COMMENTED** [review]
Hi @lusekayz , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lusekayz** [comment]
@Wes-Isaac thanks a lot. Learnt my lesson. Bless you

**Comment by Wes-Isaac** [comment]
now you can submit,

**Pull request merged**